src/csvcubedmodels/rdf/resource.py

```diff
@@ -198,9 +203,28 @@ def map_resource_to_uri(entity: RdfResource) -> URIRef:
 def map_to_literal_with_datatype(datatype: URIRef) -> Callable[[Any], Literal]:
     return lambda val: Literal(val, datatype=datatype)
 
+def map_str_to_markdown(s: str, log_html_warning: bool = True) -> Literal:
+    class ContainsHtmlParser(HTMLParser):
+        contains_html: bool
+
+        def __init__(self):
+            HTMLParser.__init__(self)
+            self.contains_html = False
+
+        def handle_starttag(self, tag, attrs):
+            self.contains_html = True
+
+        def handle_endtag(self, tag):
+            self.contains_html = True
 
-def map_str_to_markdown(s: str) -> Literal:
-    return Literal(s, MARKDOWN)
+    parser = ContainsHtmlParser()
+    parser.feed(s)
+    if parser.contains_html == True:
```

Comment:
I'd generally recommend `if parser.contains_html:` here since it means the same thing and is terser.